### DISCUSSION TITLE
Not Able to Install Podman on RHEL 8

### DISCUSSION THREAD

**Discussion by HarunMbaabu** [containers/podman#22586] [General]
Hello Engineers. I am trying to install Podman on  RHEL 8 but am getting  the following error when i run  "sudo dnf install container-tools"

**Error:** 
```
No match for argument: container-tools
Error: Unable to find a match: container-tools 
```
 
**Here is the container-tools.repo file below:** 
 
**container-tools.repo:**
```
[podman_container-tools]
name=Podman Container Tools
baseurl=https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable:/podman:/2.0/RHEL_8/
enabled=1
gpgcheck=1
gpgkey=https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable:/podman:/2.0/RHEL_8/repodata/repomd.xml.key
```
 
 Is there anyone free to help me resolve the issue?

**Comment by zackman0010** [comment]
@HarunMbaabu container-tools in RHEL 8 is a module, so you have to install it with `module install` instead of just `install`.
`sudo yum module install container-tools`
